tests/tf-manifests/aws/vpc/main.tf

```diff
@@ -53,7 +53,13 @@ module "vpc" {
 
   azs             = local.azs
   private_subnets = local.private_subnets
+  private_subnet_tags = {
```

Comment:
The tags is not required for classic. To avoid confusion, let's put a variable to control if we want to tag it.